golang/internal/grpc/grpc.go

```diff
@@ -62,16 +65,30 @@ type ContainerWatchContext struct {
 	Error  chan error
 }
 
+type UpdateOptions struct {
+	UpdateAlways  bool
+	UseContainers bool
+}
+
+type ClientLoop struct {
+	Ctx         context.Context
+	WorkerFuncs WorkerFunctions
+	Secrets     config.SecretStore
+	cancel      context.CancelFunc
```

Comment:
Cancel?

Reply:
That's for canceling the grpc call, thus the loop itself. What would you prefer?